framework/mesh/mesh_continuum/mesh_continuum.cc

```diff
@@ -15,7 +15,9 @@
 #include <vtkExodusIIWriter.h>
 #include <vtkPointData.h>
 #include <vtkCellData.h>
+
```

Comment:
remove this line, `#include`s should be in a single block.

Reply:
Done. My latest commit reformatted and reordered all of the include blocks.